docs/installation.rst

```diff
@@ -19,6 +19,27 @@ You can also install it into a new virtual environment:
    $ source ~/.virtualenvs/cernopendata-client/bin/activate
    $ pip install cernopendata-client
 
+In order to use Pycurl / xrootd related functionality, additional
+dependencies are required.
+
+The ``pycurl`` "extra" install flavor
+-------------------------------------
+
+If you're installing via ``pip`` (recommended), you should be able to get the
+optional Python package requirements by changing your installation to refer to
+``cernopendata-client[pycurl]`` (from simply ``cernopendata-client``), e.g.::
+
+    pip install "cernopendata-client[pycurl]"
```

Comment:
Needs leading `$ ...` sign to indicate shell, so that it would go nicely with the other examples mentioned on that page.

(But see also the content changes suggested above.)

Reply:
Done 👍